apps/extension/src/pages/sign/cosmos/adr36.tsx

```diff
@@ -108,6 +109,11 @@ export const SignCosmosADR36Page: FunctionComponent = observer(() => {
     Error | undefined
   >(undefined);
 
+  const isKeystonUSB =
```

Comment:
I seems that this is typo of `isKeystoneUSB`

Reply:
fixed